src/services/projects.js

```diff
@@ -125,53 +166,50 @@ module.exports = class ProjectsHelper {
 
 			bodyData = _.omit(bodyData, ['review_type', 'type', 'organization_id'])
 
-			let fileName = loggedInUserId + resourceId + orgId + 'project.json'
-
-			let getSignedUrl = await filesService.getSignedUrl(
-				{ [resourceId]: { files: [fileName] } },
-				common.PROJECT,
-				loggedInUserId
-			)
-
-			let config = {
-				method: 'put',
-				maxBodyLength: Infinity,
-				url: getSignedUrl.result[resourceId].files[0].url,
-				headers: {
-					'Content-Type': 'multipart/form-data',
-				},
-				data: JSON.stringify(bodyData),
-			}
-
-			let projectUploadStatus = await axios.request(config)
-			if (projectUploadStatus.status == 200 || projectUploadStatus.status == 201) {
+			//upload to blob
+			const fileName = `${loggedInUserId}${resourceId}${orgId}project.json`
+			const projectUploadStatus = await this.uploadToCloud(fileName, resourceId, loggedInUserId, bodyData)
+			if (
+				projectUploadStatus.result.status == httpStatusCode.ok ||
+				projectUploadStatus.result.status == httpStatusCode.created
+			) {
 				let filter = {
 					id: resourceId,
 					organization_id: orgId,
 				}
+
 				let updateData = {
-					meta: { title: bodyData.title },
-					title: bodyData.title,
 					updated_by: loggedInUserId,
-					blob_path: getSignedUrl.result[resourceId].files[0].file,
+					blob_path: projectUploadStatus.blob_path,
```

Comment:
While creating the project we are not updating blob path because the resource id is required to generate file name and path. hence after creating we are updating the db